contracts/CErc20.sol

```diff
@@ -1,18 +1,35 @@
 pragma solidity ^0.5.16;
 
 import "./CToken.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
 
 interface CompLike {
   function delegate(address delegatee) external;
 }
 
+interface RibbonMinter {
+  function mint(address gauge_addr) external;
+}
+
+interface RewardsDistributor {
+  function burn(uint256 amount) external;
+}
+
 /**
  * @title Compound's CErc20 Contract
  * @notice CTokens which wrap an EIP-20 underlying
  * @dev This contract should not to be deployed on its own; instead, deploy `CErc20Delegator` (proxy contract) and `CErc20Delegate` (logic/implementation contract).
  * @author Compound
  */
 contract CErc20 is CToken, CErc20Interface {
+    // Minter contract for rbn gauge emissions
+    RibbonMinter public constant RBN_MINTER = RibbonMinter(0x5B0655F938A72052c46d2e94D206ccB6FF625A3A);
+    // RBN token
+    IERC20Upgradeable public constant RBN = IERC20Upgradeable(0x6123b0049f904d730db3c36a31167d9d4121fa6b);
```

Comment:
We can use IERC20 because we're not touching any of the upgradeable functionality